config/environments/development.rb

```diff
@@ -13,18 +13,14 @@
   config.consider_all_requests_local = true
 
   # Enable/disable caching. By default caching is disabled.
-  if Rails.root.join('tmp/caching-dev.txt').exist?
```

Comment:
This turns caching on

Reply:
Ooo nice, so there is built in caching code we just have to implement?

Reply:
Yep, rails is hookup up to do caching, we just have to turn off the blocker by deleting a few lines of code